.github/workflows/main.yml

```diff
@@ -28,12 +28,22 @@ jobs:
         id: exclude_files
         run: (test -f ".github/workflows/blacklist.${{steps.type.outputs.type}}.txt" && echo ::set-output name=exclude_files::"($(cat ".github/workflows/blacklist.${{steps.type.outputs.type}}.txt" | tr '\n' '|'))" ) || (echo ::set-output name=exclude_files::" ")
      
-      - name: Docker 
+      - if: ${{ matrix.verify_flag != 'klee' }}
+        name: Docker 
         run: docker build -t verify-c-common . --file docker/verify-c-common.Dockerfile
+
+      - if: ${{ matrix.verify_flag == 'klee' }}
+        name: Docker with KLEE
+        run: docker build -t verify-c-common . --file docker/verify-c-common-klee.Dockerfile
```

Comment:
how expensive is this step? Perhaps we should only run klee on a nightly schedule. not for every PR

Reply:
We need a nightly action to create the container with klee, and then use pre-build container in PR builds